Source/Phone/Call/CallManager.swift

```diff
@@ -62,9 +62,10 @@ class CallManager {
         return nil
     }
     
-    func fetchActiveCalls() {
+    func fetchActiveCalls(deviceRegistrationInformation: DeviceRegistrationInformation) {
```

Comment:
There's a gradual refactor that needs to take place to make all the classes less dependent on the deviceService for data storage. We've got the registration information coming back as a clean immutable object and I'd like to see it start to be extracted from "semi-global" data. Way better would be to get the construction of the objects to take it after device registration is complete. We're not there yet, but that's the motivation here.